.github/workflows/main.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     # we can add more versions of node.js in the future
     strategy:
       matrix:
-        python-version: ['3.6', '3.7', '3.8']
+        python-version: ['3.9']
```

Comment:
FWIW the current python version of the qiita conda environment is 3.9.

Reply:
Correct. The version of the interpreter used on qiita-rc and qiita for this plugin is currently 3.9. Originally I had 3.10 and 3.11 in that matrix as well, but there are import path changes between 3.9 and 3.10 that need to be addressed in some of our dependencies for it to work. Hence, just 3.9 for now.